frontend/components/elements/StripeSetup.tsx

```
@@ -25,7 +61,21 @@ export default function StripeSetup(props: { userId: string; setLoading: Dispatc
           router.push(link);
         }}
       >
-        Register
+        {userLoading ? (
+          <span className="flex items-center gap-2" aria-busy="true">
+            <svg className="animate-spin h-4 w-4" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)